RecyclerView/app/src/main/java/net/boondockradio/recyclerview/StatesAdapter.java

```diff
@@ -74,11 +103,14 @@ public int getItemCount() {
     }
 
     class ItemViewHolder extends RecyclerView.ViewHolder {
+
         public TextView textView;
         public ImageView imageView;
+        public View itemView;
```

Comment:
Did you need to declare this? I think it's already accessible through `ViewHolder`.